yarn.lock

```diff
@@ -1688,6 +1688,17 @@
   dependencies:
     lodash "^4.17.11"
 
+"@ovh-ux/ng-ovh-cloud-universe-components@^1.2.0-alpha.0":
```

Comment:
Seems not related to the `ovh-api-services` bump

Reply:
Please refer to this Pull request first: #494

Reply:
I'll rebase and update once https://github.com/ovh-ux/manager/pull/494 is merged

Reply:
It is now merged:)